spid-spring-integration/src/test/java/it/italia/developers/spid/integration/service/test/SPIDIntegrationServiceTest.java

```diff
@@ -80,7 +80,7 @@ public void shouldGetIdWhenCallAuthRequest() throws Exception {
 			softly.assertThat(resultElement.getAttributes().getNamedItem("Destination").getTextContent()).contains(IDPEntityId);
 			softly.assertThat(resultElement.getAttributes().getNamedItem("AssertionConsumerServiceIndex").getTextContent()).isEqualTo(String.valueOf(assertionConsumerServiceIndex));
 			softly.assertThat(resultElement.getAttributes().getNamedItem("ProtocolBinding").getTextContent()).isEqualTo("urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST");
-			softly.assertThat(resultElement.getAttributes().getNamedItem("IsPassive")).isNull();
+			softly.assertThat(resultElement.getAttributes().getNamedItem("IsPassive")).isNotNull();
```

Comment:
@FaberNa mi pareva di aver capito (anche da questa guida [qui](https://docs.italia.it/italia/spid/spid-regole-tecniche/it/stabile/single-sign-on.html)) che l'attributo IsPassive non dovesse essere presente.

@peppelinux tu sei sicuramente più informato.

Reply:
- nell’elemento <AuthnRequest> non deve essere presente l’attributo IsPassive (ad indicare false come valore di default) ... in verità andrebbe aggiunto che se presente deve essere false

Reply:
un giro con spid-sp-test ci darebbe contezza sulle criticità, esempio:

https://github.com/italia/spid-sp-test/blob/8b8dc56884cfcdb2a218e975a4738318ef87f408/src/spid_sp_test/authn_request.py#L523

Questi sono i controlli che AgID fa in fase di collaudo